source/__tests__/integration/jestPuppeteerHelper.js

```diff
@@ -0,0 +1,55 @@
+async function expectSelTextContentToMatch(sel, expected) {
+  await page.waitForSelector(sel)
+  const textContent = await page.$eval(sel, e => e.textContent)
+  return expect(textContent).toMatch(expected)
+}
+async function expectSelTextContentNotToMatch(sel, expected) {
+  const textContent = await page.$eval(sel, e => e.textContent)
+  return expect(textContent).not.toMatch(expected)
+}
+async function expectSelTextContentTrimmedToMatch(sel, expected) {
+  await page.waitForSelector(sel)
+  const textContent = await page.$eval(sel, e => e.textContent)
+  return expect(textContent.trim()).toMatch(expected)
+}
+async function expectSelValueToBe(sel, expected) {
+  await page.waitForSelector(sel)
+  const value = await page.$eval(sel, e => e.value)
+  return expect(value).toBe(expected)
+}
+async function expectSelTextContentToBe(sel, expected) {
+  await page.waitForSelector(sel)
+  const textContent = await page.$eval(sel, e => e.textContent)
+  return expect(textContent).toBe(expected)
+}
+async function expectSelTextContentNotToBe(sel, value) {
+  await page.waitForSelector(sel)
+  const textContent = await page.$eval(sel, e => e.textContent)
+  return expect(textContent.trim()).not.toBe(value)
+}
+async function expectSelTextContentTrimmedToBe(sel, expected) {
+  await page.waitForSelector(sel)
+  const textContent = await page.$eval(sel, e => e.textContent)
+  return expect(textContent.trim()).toBe(expected)
+}
+async function expectSelToExist(sel) {
+  await page.waitForSelector(sel)
+  const selector = !!(await page.$(sel))
```

Comment:
huh, it is weird to me that this is (apparently) idiomatic javascript (as opposed to e.g. `new Boolean((await page.$(sel)))` or `(await page.$(sel)) == true`, but whatever.

Reply:
also i might call it `isSelectorPresent` or something more boolean-ish, but certainly wouldn't hold up merge over it.

Reply:
that would be a better name.

Reply:
will put in my current PR.

Reply:
thx!